aten/src/ATen/native/BatchLinearAlgebraKernel.cpp

```diff
@@ -794,50 +805,46 @@ This is an in-place routine, content of 'B' is overwritten.
 'transpose' if true then op(A) = A^T,
 'unitriangular' if true then the diagonal elements of A are assumed to be 1
 and the actual diagonal values are not used.
-'infos' is an int Tensor containing error codes for each matrix in the batched input.
-For more information see LAPACK's documentation for TRTRS routine.
 */
 template<typename scalar_t>
-void apply_triangular_solve(Tensor& A, Tensor& B, Tensor& infos, bool upper, bool transpose, bool conjugate_transpose, bool unitriangular) {
+void apply_triangular_solve(Tensor& A, Tensor& B, bool left, bool upper, TransposeType transpose, bool unitriangular) {
 #if !AT_BUILD_WITH_LAPACK()
   TORCH_CHECK(
       false,
       "Calling torch.triangular_solve on a CPU tensor requires compiling ",
       "PyTorch with LAPACK. Please use PyTorch built with LAPACK support.");
 #else
```

Comment:
With changes in this PR, LAPACK is not needed anymore for this function. It uses BLAS.

Reply:
`#if !AT_BUILD_WITH_BLAS()` should be used here.